test/metabase/api/dashboard_test.clj

```diff
@@ -2600,10 +2596,6 @@
             (let [execute-path (format "dashboard/%s/dashcard/%s/execute"
                                        dashboard-id
                                        dashcard-id)]
-              (testing "Without actions enabled"
-                (is (= "Actions are not enabled."
-                       (mt/user-http-request :crowberto :post 400 execute-path
-                                             {:parameters {"id" 1}}))))
```

Comment:
Likewise here. We still need to test if actions are not enabled on the database. I tested this one, and it actually lets you execute the action even when actions aren't enabled on the database. But only for `:type :query` and `:type :http` actions.

I checked on master and found this is an existing bug. It just wasn't tested before. We could even fix this separately to this PR. But this test shouldn't be removed.

For example, this test passes when it should return a 400:
```
(deftest dashcard-implicit-action-execution-auth-test
  (mt/with-temp-copy-of-db
    (actions.test-util/with-actions-test-data
      (testing "Executing dashcard with action"
        (actions.test-util/with-actions [{:keys [action-id model-id]} {}] <--- note this is `:type :query` by default
          (mt/with-temp* [Dashboard [{dashboard-id :id}]
                          DashboardCard [{dashcard-id :id} {:dashboard_id dashboard-id
                                                            :action_id action-id
                                                            :card_id model-id}]]
            (let [execute-path (format "dashboard/%s/dashcard/%s/execute"
                                       dashboard-id
                                       dashcard-id)]
              (testing "Without actions enabled"
                (is (= {:rows-affected 1}
                       (mt/user-http-request :crowberto :post 400 execute-path
                                             {:parameters {"id" 1}})))))))))))
```